perfkitbenchmarker/linux_packages/netperf.py

```diff
@@ -15,13 +15,25 @@
 
 """Module containing netperf installation and cleanup functions."""
 
+import re
+
+from perfkitbenchmarker import flags
 from perfkitbenchmarker import vm_util
 
+flags.DEFINE_integer(
+    'netperf_histogram_buckets', 100,
+    'The number of buckets per row in the internal histogram that netperf '
```

Comment:
Maybe have a more details about row. "Netperf internal histogram" is not very obvious.